app/src/main/java/ru/appintheair/flightreviewssubmit/SubmitScreen.kt

```diff
@@ -0,0 +1,167 @@
+package ru.appintheair.flightreviewssubmit
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.view.LayoutInflater
```

Comment:
Text field is not visible under the keyboard
[video.mp4.zip](https://github.com/Pank-Off/FlightsApp/files/5537963/video.mp4.zip)